packages/components-vue/index.html

```diff
@@ -9,8 +9,8 @@
+    <!--PLACEHOLDER_PORSCHE_DESIGN_SYSTEM_FONT_LATIN--><link rel=preload href=http://localhost:3001/fonts/porsche-next-w-la-regular.min.b8f1c203bd2ac5334247025236ba8d86.woff2 as=font type=font/woff2 crossorigin><link rel=preload href=http://localhost:3001/fonts/porsche-next-w-la-semi-bold.min.b5f6fca7225aa73578f65320bcf79b45.woff2 as=font type=font/woff2 crossorigin><link rel=preload href=http://localhost:3001/fonts/porsche-next-w-la-bold.min.0fbdc6d73f04137fff6e4169e8c30e2c.woff2 as=font type=font/woff2 crossorigin>
```

Comment:
Regression with the hashes?

Reply:
Not a regression, its the same hash as in our components-js index html. The replace script is only triggered via prebuild-app which nobody has done for quite some time.